RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py

```diff
@@ -3,5 +3,12 @@
 pfDeepCSVJetTags = cms.EDProducer(
 	'DeepFlavourJetTagsProducer',
 	src = cms.InputTag('pfDeepCSVTagInfos'),
+  checkSVForDefaults = cms.bool(False),
+  meanPadding = cms.bool(False),
 	NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepFlavourNoSL.json')
 	)
+
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+phase1Pixel.toModify(pfDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
```

Comment:
Can json files such as this one used in the DeepCSV tagger be put in the CondDB? We can put GBRForest payloads in the CondDB for BDT-based taggers but for a general MVA technique, I'm not sure what are our options (of course, there are conditions used for the CSV algorithm based on a dedicated MVA computer code in `PhysicsTools/MVAComputer` but the general trend is to move away from that code since it is not very user-friendly and there are no longer developers around who really understand the inner workings of the code).